src/js/route.js

```diff
@@ -15,10 +12,15 @@ Router.prototype.toString = function() {
 };
 
 Router.prototype.constructDomain = function() {
-    if (this.absolute)
+    if(this.name === undefined) {
```

Comment:
@Robert-Hansen 
Any reason behind removing check for absolute ?
```js
if (this.absolute)
```
https://github.com/tightenco/ziggy/blob/master/src/js/route.js#L18